woocommerce/woocommerce-android
# [TTP] Testing TTP makes other payment methods disabled

Closes: #9408


### Description

The PR disables all the rows on the payment selection screen when TTP testing flow is used. Also, it brings small UI improvements to this screen.

### Testing instructions

* Run Order, simple payments and TTP testing flow
* Use both dark and light mode
* Notice that during TTP testing flow all payment methods are disabled except TTP


### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/b7b29a5e-8248-48ee-a913-cddd6a8cbeb6
